### DISCUSSION TITLE
How to use my own revanced patches

### DISCUSSION THREAD

**Discussion by pranavxmeta** [ReVanced/revanced-discussions#814] [Question] [answered]
I have forked this repo but how do I access patches in revanced manager? 

Changing the patches organization to my username (thegravity98) doesn't work. It stays stuck on logo screen.

**Comment by oSumAtrIX** [comment, marked as answer]
If you forked the patches repository you will need to create a release by running the workflow `Release`.

**Reply by pranavxmeta** [reply]
Thanks

**Reply by andybc262** [reply]
I'm noob, how can I do this?

**Reply by oSumAtrIX** [reply]
Check out the GitHub docs on how to use GitHub

**Reply by andybc262** [reply]
i've forked and released but revanced manager stucks in 

Initializing
creating working directory
